static/css/home.css

```diff
@@ -0,0 +1,278 @@
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+}
+#features > ul > li {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 6rem 2rem;
+}
+#features > ul > li > ul > li{
+  margin: 0 0 1rem;
+}
+#features > ul > li:nth-child(2) {
+  background-color: #fafafa;
+}
+#features .action-button {
+  margin: 4rem 9rem 1rem;
+}
+
+.articles {
+  padding: 6rem 0;
+  background-color: #fafafa
+}
+.articles .preview {
+  margin: 2rem;
+}
+.articles .row .preview a h1 {
+  min-height: 8rem;
+  display: flex;
+  align-items: center;
+  font-size: 1.4em;
+}
+.articles article figure {
+  height: 25rem;
+}
+.read.action-button {
+  width: 50%;
+  margin: 0 auto 0.5rem;
+}
+
+footer {
+  position: relative;
+  top: -9rem;
+}
+
+@media screen and (max-width:1200px) {
+  section:first-child  > div > h1 {
+    top: 16rem;
+    left: 8rem;
+  }
+  section:first-child  > div > h2 {
+    top: 25rem;
+    left: 8rem;
+  }
+  #join-us {
+    top: 31rem;
+    left: 11.65rem;
+  }
+  #join-us.sticky {
+    left: 18rem;
+  }
+  .column {
+    flex-direction: column;
+    align-items: center;
+    padding: 0 22rem;
+  }
+  .articles .column .preview a h1 {
+    min-height: auto;
+  }
+}
+
+@media screen and (max-width:960px) {
+  section:first-child  > div > h1 {
+    top: 10rem;
+    left: 3.5rem;
+  }
+  section:first-child  > div > h2 {
+    top: 20rem;
+    left: 3.5rem;
+    max-width: 36rem;
+    font-size: 2.4rem;
+    line-height: 3rem;
+  }
+  #join-us {
+    top: 29rem;
+    left: 7.15rem;
+  }
+  .column {
+    padding: 0 10rem;
+  }
+}
+
+@media screen and (max-width:620px) {
+  body > header {
+    padding: 0;
+  }
+  #join-us.sticky {
+    top: 4.5rem;
+    left: 0;
+    right: 0;
+    text-align: center;
+    background-color: #279e6aee;
+    border: none;
+    font-weight: 300;
+    border-radius: 0;
+  }
+  #join-us.sticky::after {
+    font-family: FontAwesome;
+    content: "\f0a5";
+    font-size: 2rem;
+    color: #fff;
+    position: relative;
+    left: 2rem;
+  }
+  #features .action-button {
+    margin: 4rem 3rem 0rem;
+  }
+  .column {
+    padding: 0 5rem;
+  }
+  .articles article figure {
+    height: auto;
+  }
+}
+
+@media screen and (max-width:480px) {
+  section:first-child > div > h1 {
+    font-size: 7.5rem;
+    top: 9rem;
+    left: 2rem;
+  }
+  section:first-child > div > h2 {
+    font-size: 2rem;
+    top: 18rem;
+    left: 2rem;
+    width: 30rem;
+  }
+  #join-us {
+    top: 27rem;
+    left: 5.65rem;
+  }
+  .articles {
+    padding: 5rem 0;
+  }
+  .column {
+    padding: 0 5rem;
+  }
+}
+
+@media screen and (max-width:360px) {
+  section:first-child > div > h1 {
+    font-size: 5.5rem;
+    margin: 0;
+  }
+  section:first-child  > div > h2 {
+    margin: 0;
+    top: 16rem;
+    left: 2.5rem;
+    width: 90%;
+  }
+  #join-us {
+    top: 25rem;
+    left: 2.45rem;
+  }
+  .column {
+    padding: 0 3.5rem;
+  }
+}
+
+@keyframes updown {
+  0%   { top: -7rem; }
```

Comment:
it looks weird to move from -7rem to -5rem.
maybe you just want to move your item to `top: -5rem` and to make the movement changing another property (like `margin-top` for example)?